webkit2/webkit2.find-controller.lisp

```diff
@@ -15,11 +15,54 @@
    ("text" "gchararray")
    ("web-view" "WebKitWebView" t t)))
 
-(defcfun "webkit_find_controller_search" :void
+(define-g-enum "WebKitFindOptions" webkit-find-options ()
+  :webkit-find-options-none
+  (:webkit-find-options-case-insensitive 1)
+  (:webkit-find-options-at-word-starts 2)
+  (:webkit-find-options-treat-medial-capital-as-word-start 4)
+  (:webkit-find-options-backwards 8)
+  (:webkit-find-options-wrap-around 16))
+
+(defcfun ("webkit_find_controller_search" %webkit-find-controller-search) :void
   (find-controller (g-object webkit-find-controller))
   (search-text :string)
   (find-options :uint)
   (max-match-count :uint))
+
+(defun webkit-find-controller-search (controller search-text
+                                      &key (max-match-count
+                                            ;; I.e., UINT_MAX
+                                            (1- (expt 2 (* 8 (cffi:foreign-type-size :uint)))))
+                                        (case-insensitive t)
+                                        at-word-starts
+                                        treat-medial-capital-as-word-start
+                                        backwards
+                                        (wrap-around t))
+  "A smarter version of webkit_find_controller_search with sensible defaults.
```

Comment:
Wouldn't this change be backward incompatible?

Reply:
Indeed it will be backwards incompatible :( Maybe I need to come up with a different name for it.